EIPS/eip-3525.md

```diff
@@ -1,54 +1,38 @@
 ---
 eip: 3525
 title: Semi-Fungible Token Standard
```

Comment:
We try to discourage the word "standard" in titles and descriptions. Every EIP ends up being a standard, so it's a bit redundant.